doc/manual/spec.md

```diff
@@ -189,12 +303,81 @@ the libraries to access an ext2 file system, you could express this as
 	BuildRequires: e2fsprofs-devel = 1.17-1
 ```
 
-Finally, if your package used C++ and could not be built with gcc-2.7.2.1, you
-can express this as
+#### Buildconflicts
+
+Capabilities which conflict, ie cannot be installed during the package
+package build.
+
+For example if somelib-devel presence causes the package to fail build,
+you would add
 ```
-	BuildConflicts: gcc <= 2.7.2.1
+	BuildConflicts: somelib-devel
 ```
 
+#### Excludearch
+
+Package is not buildable on architectures listed here.
+Used when software is portable across most architectures except some,
+for example due to endianess issues.
+
+#### Exclusivearch
+
+Package is only buildable on architectures listed here.
+For example, it's probably not possible to build an i386-specific BIOS
+utility on ARM, and even if it was it probably would not make any sense.
+
+#### Excludeos
+
+Package is not buildable on specific OS'es listed here.
+
+#### Exclusiveos
+
+Package is only buildable on OS'es listed here.
+
+#### Buildarchs
```

Comment:
`Buildarches` -> `BuildArch`

Reply:
Right, either `BuildArch` or `BuildArchitectures`, but not `BuildArchs`...